code/modules/surgery/organs/organ_internal.dm

```diff
@@ -195,7 +195,7 @@
 	return S
 
 /obj/item/organ/internal/attempt_become_organ(obj/item/organ/external/parent,mob/living/carbon/human/H)
-	if(parent_organ != parent.limb_name)
+	if(parent_organ != parent.limb_name && parent_organ != "eyes" && parent_organ != "mouth")
```

Comment:
this seems oddly specific, is there any reason why it's just eyes and mouth here, and not other organs that are internal to heads?

Reply:
Because eyes and mouth are the only two target zones that do not have a dedicated parent limb instead being apart of the head

Reply:
Also don't confuse 'eyes' here for the actual eyes organ, eyes here refer to the parent_organ, the slot in which the organ goes into. It's any organ that fits into the eyes, same with mouth. Other head organs like CNS rebooters or a brain does not need this because its paren_organ has the same name as the parent limb . . . 'head'

Reply:
Aight, got it.